src/main/java/com/jcabi/github/mock/MkUser.java

```diff
@@ -169,4 +170,14 @@ private String xpath() {
         return String.format("/github/users/user[login='%s']", this.self);
     }
 
+	@Override
+	public Iterable<Coordinates> invitations() throws IOException {
+		throw new UnsupportedOperationException();
+	}
+
+	@Override
+	public boolean acceptInvitation(Coordinates coords) throws IOException {
```

Comment:
@ShalomGottesman Can you make ``coorde`` final? Generally, it's good to work only with final variables and parameters.

Final variables are immutable: this means that an object with immutable fields is thread-safe by default and also a method that works only with final variables will be easier to understand, there will be only one initialization per variable :)